lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -468,9 +468,14 @@ newDBLayer trace fp timeInterpreter = do
                 . MsgRemovingRetiredPools
 
         readPoolProductionCursor k = do
-            reverse . map (snd . fromPoolProduction . entityVal) <$> selectList
+            pp <- reverse . map (snd . fromPoolProduction . entityVal) <$> selectList
                 []
                 [Desc PoolProductionSlot, LimitTo k]
+            case pp of
+                -- byron headers might not be inserted
+                -- continuously, so only pick the last.
+                [] -> listByronHeaders 1
+                xs -> pure xs
```

Comment:
Request: make this a bit shorter. (See [line limit](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters).)

Here's one potentially nice way:
```suggestion
            pp <- map (snd . fromPoolProduction . entityVal) <$> selectList
                []
                [Desc PoolProductionSlot, LimitTo k]
            case pp of
                -- byron headers might not be inserted
                -- continuously, so only pick the last.
                [] -> listByronHeaders 1
                xs -> pure $ reverse xs
```